website_snippet_product_category/static/src/scss/snippet.scss

```diff
@@ -0,0 +1,42 @@
+.categ_container {
+    border: 0 solid $border-color;
+    border-width: 0 1px 0 0;
+
+    .categ_tree_level {
+        &.main_tree_level {
+            color: $primary;
+            text-transform: uppercase;
+        }
+
+        &[data-tree-level="2"] {
+            font-size: 0.9em;
+            .categ_link {
+                text-transform: capitalize;
+                color: $gray-800;
+            }
+            .categ_img img {
+                width: 80%;
```

Comment:
Would it make more sense to use BS classes such as `col-10` and so on? (the same for all `width` declarations)

Reply:
Ummm.... for me try to use BS for this behaviour adds more complexity to the xml file and for alignment operations too